### PR TITLE
[Feat]: add-sound-to-notification

### PR DESCRIPTION
**Pull request by ferasbbm**

### CODE REVIEW

**Comment by gentritabazi01** [comment]
CC @samushi 

image

**Comment by ferasbbm** [comment]
New Feature:
Added sound to notification payload because The onMessageReceived method is fired only notifictaion without sound

File: src/Services/Larafirebase.php [hunk 1]
```diff
@@ -101,7 +110,9 @@ public function sendNotification($tokens)
                 'body' => $this->body,
                 'image' => $this->image,
                 'icon' => $this->icon,
+                'sound' => $this->sound,
                 'click_action' => $this->clickAction
+
```
Comment (gentritabazi01) [inline src/Services/Larafirebase.php:115]: Remove extra line here.

File: src/Services/Larafirebase.php [hunk 2]
```diff
@@ -64,6 +66,13 @@ public function withIcon($icon)
         return $this;
     }
 
+    public function withSound($sound)
+     {
+         $this->sound = $sound;
+
+         return $this;
+     }
```
Comment (gentritabazi01) [inline src/Services/Larafirebase.php:74]: Format code.

**Review by ferasbbm — COMMENTED** [review]
hi there, u can check all enhancements...Thanks!

**Review by astritzeqiri — APPROVED** [review]

**Review by gentritabazi01 — APPROVED** [review]

**Pull request merged**